modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -115,6 +115,7 @@ event(#postback{message={delete_all, Args}}, Context) ->
             z_render:wire({alert, [{message, ?__("Delete is canceled, there are users in the user groups.", Context)}]}, Context)
     end.
 
+-spec ug_delete(list(pos_integer()), #context{}) -> any().
 ug_delete(Ids, Context) ->
     z_session_page:add_script(z_render:wire({mask, [{message, ?__("Deleting...", Context)}]}, Context)),
```

Comment:
The code in the following (LINE, COL) locations has the same structure: (120, 19), (142, 19).